packages/gamepads_android/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: gamepads_android
+description: Android implementation of gamepads, a Flutter plugin to handle gamepad input across multiple platforms.
+version: 0.1.1
+homepage: https://github.com/flame-engine/gamepads
+repository: https://github.com/flame-engine/gamepads/tree/main/packages/gamepads_android
+
+flutter:
+  plugin:
+    platforms:
+      android:
+        package: dev.markvideon.gamepads_android
+        pluginClass: GamepadsAndroidPlugin
+
+environment:
+  sdk: '>=2.19.0 <4.0.0'
+  flutter: '>=3.3.0'
```

Comment:
This has a higher version constraint than `gamepads`, is that intentional?

Reply:
Nope, good catch